Bottom: 318f5961b0496b4f6465e72d05353a1572ad5602
Top:    4efd7b6c9e814f32b2c3dae0d7899908f43dc624
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-11 23:39:31 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
